WildlifeRescueCenter/Program.cs

```diff
@@ -34,12 +34,7 @@ static void Main(string[] args)
         Enclosures Outdoorenclosures = new Enclosures("Outdoor enclosures", 15);
         Enclosures Indoorenclosures = new Enclosures("Indoor enclosures", 2);
 
-        List<Enclosures> worksites = new List<Enclosures>() { Outdoorenclosures, Indoorenclosures};
-
-        int numberOfAnimal = 0;
-        int numberOfAnimalIndoor = 0;
-        int numberOfAnimalOutdoor = 0;
```

Comment:
Typicky chceme promenne definovat, aby mely co nejmensi scope, tedy co nejblize jejich pouziti. To nam take trochu zjednodussi kod, jelikoz tu promennou potom nemusime posilat dale do `CapacityOfIndoorEnclosure` 🙂